source/modulo_core/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ find_package(ament_cmake_python REQUIRED)
 find_package(geometry_msgs REQUIRED)
 find_package(sensor_msgs REQUIRED)
 find_package(std_msgs REQUIRED)
-find_package(rclcpp REQUIRED)
+find_package(rclcpp_lifecycle REQUIRED)
```

Comment:
🤨 I see why adding `rclcpp_lifecycle` dependency is a good idea, but I don't like the removal of `rclcpp`. I would prefer to have both at least (and not the implicitly take rclcpp as a dependency of lifecycle) - same for target / export dependencies

Reply:
You are totally right there